docs/user/project_metadata.md

```diff
@@ -30,21 +30,44 @@ PyPI currently supports several ways of verifying project URLs. When a URL is ve
     PyPI package owner, and does not imply any additional safety about that
     URL or any other relationship to the project in question.
 
-### Self-links
+The following subsections specify the different types of URLs that can be verified.
 
-Any URL to the project on PyPI will be considered verified.
+#### Self-links
 
-### Via Trusted Publishing
+Any URL to the project on PyPI will be considered verified. For example, the project page for `pip` will mark all of the
+following as verified:
+
+- https://pypi.org/project/pip/
+- https://pypi.org/p/pip/
+- https://pypi.python.org/project/pip
+- https://pypi.python.org/p/pip
+- https://python.org/pypi/pip
+
+#### Via Trusted Publishing
 
 [Trusted Publishing](trusted-publishers/index.md) allows PyPI to attest that the 
 publishing workflow for a package is coming from a verified source.
 
-PyPI supports the following OIDC Publishers:
+The URLs that can be verified depend on the Trusted Publisher used:
+
+- [GitHub Actions][gh-action-tab]: Packages uploaded using GHA from a repository will have the GitHub URLs for that
+  repository verified. For example, for the `pypa/pip` repository, the following URLs will be verified:
+    - https://github.com/pypa/pip
+    - https://github.com/pypa/pip/* (all subpaths)
+    - https://github.com/pypa/pip.git
+    - https://pypa.github.io/pip
+    - https://pypa.github.io/pip/* (all subpaths)
```

Comment:
Minor thing, but consider putting these (and below) in backticks -- IMO having them formatted as code better clarifies that we're not telling people to click on them.